web_server/posts/models.py

```diff
@@ -36,19 +45,18 @@ class Post(models.Model):
     author = models.ForeignKey(Author, on_delete=models.PROTECT)
 
     # Storing a list of comma delineated categories
-    # @todo Should this be replaced with a many-to-many combo table and a separate 'categories' model?
-    # categories = models.CharField(max_length=256)
+    categories = models.ManyToManyField(Category, blank=True)
 
     # Count of comments
+    # @todo is this field strictly necessary? It should be computed not stored
```

Comment:
You are right, it should be computed during the get requests. It can be done easily if we have comments as a manytomany model in the post model.

Reply:
I've removed the field and left a note